platforms/ios/DEVELOPING.md

```diff
@@ -0,0 +1,26 @@
+Developing the Tangram iOS SDK
+==============================
+
+This page is for those who intend to modify Tangram ES itself, not just use it in another application.
+
+## Code signing changes should not be committed ##
+
+When building and running the demo application on an iOS device, you will likely need to modify the "App ID" and "Development Team" values in the Xcode project in order to sign the application package correctly because Apple mandates that an App ID can only be used by one Development Team. This is a good security practice, but since the App ID and Developer Team values are embedded in the Xcode project data, there is no way for each developer working with the repository to use their App ID / Developer Team values without modifying the Xcode project files that are checked into source control.
+
+Unfortunately, this means that developers will need to either:
+  1. Consistently ignore these changes when checking git status, or
+  2. Frequently restore the Xcode project from git and re-enter the App ID / Developer Team changes as needed.
+
+## Builds must be executed from the Xcode workspace ##
+
+The Xcode workspace that contains both the demo app project and the framework project uses specific build location settings to ensure that build products for both the framework and demo app are placed in the right locations and are able to build together successfully. If you try to open the projects separately and build them, the build output might not be in the expected location or it might not succeed at all.
```

Comment:
~I am not sure if the language above is correct based on the apple documentation (link you mentioned below)~

~Based on the documentation, it seems one can open the framework project explicitly and build it and it should be fine.~


Hmm, for clarity maybe change `build output` to `build directory`, and remove "might not succeed at all". 
> You can use the workspace’s default build directory or you can specify one. Note that if a project specifies a build directory, that directory is overridden by the build directory of whatever workspace the project is in at the time you build the project.

Reply:
True, I think the framework project would be fine by itself, but the demo app project could either fail to build or link to a "stale" framework. I'll update the text to reflect that.